src/groups/mwc/mwcc/mwcc_orderedhashmap.h

```diff
@@ -296,6 +296,9 @@ class OrderedHashMap_SequentialIterator {
 
     // MANIPULATORS
 
+    /// Assign to this object the value of the specified `rhs` object.
+    OrderedHashMap_SequentialIterator& operator=(const NcIter&);
```

Comment:
Nit: Because of the documentation comment, I think we want an `rhs` name here (like in the other commit in this PR):
`OrderedHashMap_SequentialIterator& operator=(const NcIter& rhs);`

Reply:
Oh, I already force-pushed a fix for this. Might be GH lagging

Reply:
Huh, you're right!  Looks good, go ahead and merge.